asv_bench/benchmarks/parser_vb.py

```diff
@@ -1,71 +1,49 @@
 from .pandas_vb_common import *
 import os
```

Comment:
yep, it was still on my to do list to consolidate the benchmarks for the io functions (as you now have `parsers_vb.py`, but also `io_bench.py`, `hdfstore_bench.py`, `io_sql.py`). But good idea to call this one `parser.py`. Or maybe `io_parser.py` for consistency with the others (and then all io related ones are a bit grouped in the file list)